checker/tests/run-pass/test_source.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+#[macro_use]
+extern crate mirai_annotations;
+
+//=TEST_ONLY
+//=NO_ANGELIC
+
+#[test]
+fn some_test() {
+    verify!(1 == 1);
+}
+
+#[test]
+fn another_test() {
+    verify!(2 == 1); //~provably false verification condition
+}
+
+#[test]
+fn no_summary_analyzed_anyway() {
+    trait Dynamic {
+        fn f(&self, x: u64) -> u64;
+    }
+    struct S;
+    impl Dynamic for S {
+        fn f(&self, x: u64) -> u64 {
+            return x + 1;
+        }
+    }
+    let d: &dyn Dynamic = &S {} as &dyn Dynamic; // forget type info of S
+    verify!(d.f(1) == 3); // in angelic mode, this statement would disable verification for this
+                          // function and we would not see the message below. With no_angelic,
+                          // we do not see an error here (as d.f is uninterpreted), but we still
+                          // see the below.
+    verify!(3 == 4); //~provably false verification condition
+}
+
+#[allow(dead_code)]
```

Comment:
Or just make the function public, in which case the property you are testing will actually be tested. As it is now, there won't an error message anyway for non public functions.